RELEASE.md

```diff
@@ -0,0 +1,308 @@
+# Release Procedure
+
+## Table of contents
+
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Open a Release Pull Request](#open-a-release-pull-request)
+- [Bump the Version](#bump-the-version)
+- [Pick a Release Name](#pick-a-release-name)
+- [Generate Reference Tests](#generate-reference-tests)
+  - [Install Git Large File Storage](#install-git-large-file-storage)
+  - [Prepare Tests Repository](#prepare-tests-repository)
+  - [Commit Reference Tests](#commit-reference-tests)
+  - [Bundle Reference Tests](#bundle-reference-tests)
+- [Create Tests Release](#create-tests-release)
+- [Create Specs Release](#create-specs-release)
+- [Click the Release Buttons](#click-the-release-buttons)
+- [Make an Announcement](#make-an-announcement)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This document describes the necessary steps to produce a consensus-specs release.
+
+## Open a Release Pull Request
+
+> [!NOTE]
+> Try to do this at least a few days prior to the release.
+
+First, create a PR which merges `dev` into `master`.
+
+> [!TIP]
+> Click on the following link to draft a PR:
+> * https://github.com/ethereum/consensus-specs/compare/dev...master?expand=1
+
+Title this PR "Release &lt;version&gt;" (_e.g._, "Release v1.5.0-alpha.10").
+
+In the PR's description, provide a list of items that must be done. At the bottom, list unmerged PRs
+which are to be included in the release; this signals to other maintainers and developers that they
+should review these PRs soon.
+
+```markdown
+- [ ] testgen
+- [ ] version bump
+- [ ] #1234
+- [ ] #2345
+- [ ] #3456
+```
+
+## Bump the Version
+
+Next, update the `VERSION.txt` file which contains the eth2spec version.
+
+> [!TIP]
+> Click on the following link to open the GitHub editor for this file:
+> * https://github.com/ethereum/consensus-specs/edit/dev/tests/core/pyspec/eth2spec/VERSION.txt
+
+Next, change the version to the appropriate value and click the "Commit changes..." button.
+
+For the commit message, put "Bump version to &lt;version&gt;" (_e.g._, "Bump version to 1.5.0-alpha.10").
+
+Next, click the "Propose changes" button and proceed to make the PR.
+
+## Pick a Release Name
+
+Generally, names are based on some theme. For example, for Electra, releases are named after
+Electric-type Pokemon.
+
+> [!NOTE]
+> Please ensure that the name you choose does not have an unwanted meaning in other languages;
+use [WordSafety.com](http://wordsafety.com) to check this.
+
+## Generate Reference Tests
```

Comment:
you could also just use the CI to generate the test vectors btw: https://github.com/ethereum/consensus-specs/actions/workflows/generate_vectors.yml